### PR TITLE
Add example block content 

### PR DESCRIPTION
**Pull request by richtabor**
With the new block inserter help panel in place and the example block content API layer in progress, we can now add neat little previews of our blocks (https://github.com/WordPress/gutenberg/issues/16979). 

The Gutenberg plugin is necessary to see the previews. 

This PRs takes a first stab at adding examples to _most_ of our blocks. The innerBlocks examples are still being fleshed out, but this PR focuses on all our other blocks. Once innerBlocks support makes it into the Gutenberg plugin, we should do a second pass to add tailored content to our nested blocks.

Gif:
ScreenFlow

### CODE REVIEW

File: src/blocks/gallery-carousel/index.js [hunk 1]
```diff
@@ -39,6 +39,17 @@ const settings = {
 	supports: {
 		align: [ 'wide', 'full' ],
 	},
+	example: {
+		attributes: {
+			gridSize: 'lrg',
+			gutter: 5,
+			images: [
+				{ url: '/wp-content/plugins/coblocks/dist/images/examples/gallery-1.jpg' },
```
Comment (richtabor) [inline src/blocks/gallery-carousel/index.js]: One  bit I'm not 100% about is how I'm pulling images from the plugin to use in the examples for the gallery blocks. Thoughts?

Comment (jrtashjian) [reply src/blocks/gallery-carousel/index.js]: I think this is perfectly fine. We're not adding too many images and the file sizes are not large at all. I would prefer loading local images over potentially breaking the experience if a remote image fails to load.

File: src/blocks/author/index.js [hunk 2]
```diff
@@ -28,6 +28,13 @@ const settings = {
 	description: __( 'Add an author biography to build credibility and authority.' ),
 	icon,
 	keywords: [ __( 'biography' ), __( 'profile' ), 'coblocks' ],
+	example: {
+		attributes: {
+			name: 'Jane Doe',
+			biography: __( 'Born to express, not to impress. A maker making the world I want.' ),
+			imgUrl: '/wp-content/plugins/coblocks/dist/images/examples/author.jpg',
```
Comment (mtias) [inline src/blocks/author/index.js]: This might break if the user has specified a different wp-content folder.

Comment (gziolo) [reply src/blocks/author/index.js]: Unless base URL is properly set in the head of HTML document.

Comment (jrtashjian) [reply src/blocks/author/index.js]: Good call. Looks like we need to expose the url produced through [`plugins_url()`](https://developer.wordpress.org/reference/functions/plugins_url/) in the admin to create the proper URL here.

Comment (gziolo) [reply src/blocks/author/index.js]: This is how webpack can handle public paths:

https://webpack.js.org/guides/public-path/

If you will find it useful, we can add it to wp-scripts build, too.

Comment (jrtashjian) [reply src/blocks/author/index.js]: I'm still planning to figure this out so we can server images locally. For now we're working on using an external source for now in #1106.

**Pull request merged**